api/models/team.js

```diff
@@ -1,32 +1,28 @@
-const sequelize = require('../../db/sequelize')
-const {Sequelize, Model, DataTypes} = require('sequelize');
+const { Sequelize } = require('sequelize');
+const db = require('../db')
 
-
-module.exports = sequelize.define('team',{
-	id: {
-		field: 'teamId',
-		type: Sequelize.INTEGER,
-		primaryKey: true,
-		autoincrement: true
-	},
-	Abbreviation: {
-		field: 'abbr',
-		type: Sequelize.STRING
-	},
-	City: {
-		field: 'city',
-		type: Sequelize.STRING
-	},
-	Nickname: {
-		field: 'nickname',
-		type: Sequelize.STRING
-	}
+module.exports = db.connection.define('team',
+	{
```

Comment:
It seems that a lot of the preexisting files are stuck in "tab" mode, rather than spaces. I didn't spend a lot of time trying to fix this, but we eventually want to figure it out.